Update version of zone.js for Angular 4.0.0-rc.4

<!--
IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION YOUR ISSUE MIGHT BE CLOSED WITHOUT INVESTIGATING
-->
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```


### Versions.
<!--
Output from: `ng --version`.
If nothing, output from: `node --version` and `npm --version`.
  Windows (7/8/10). Linux (incl. distribution). macOS (El Capitan? Sierra?)
-->


### Repro steps.
<!--
Simple steps to reproduce this bug.
Please include: commands run, packages added, related code changes.
A link to a sample repo would help too.
-->
Create a new project
`ng new foo --ng4`
Ensure version of @angular/core
`npm ls @angular/core`


### The log given by the failure.
```bash
<!-- Normally this include a stack trace and some more information. -->
└── UNMET PEER DEPENDENCY zone.js@0.7.8

npm ERR! peer dep missing: zone.js@^0.8.4, required by @angular/core@4.0.0-rc.4
```


### Desired functionality.
<!--
What would like to see implemented?
What is the usecase?
-->
Update the version of zone.js to the version above for apps created using Angular 4.0.0-rc.4 or later.

--- a/packages/@angular/cli/blueprints/ng/files/package.json
+++ b/packages/@angular/cli/blueprints/ng/files/package.json
@@ -22,7 +22,7 @@
     "@angular/router": "<%= ng4 ? '>=4.0.0-beta <5.0.0' : '^3.4.0' %>",
     "core-js": "^2.4.1",
     "rxjs": "^5.1.0",
-    "zone.js": "^0.7.6"
+    "zone.js": "^0.8.4"
   },
   "devDependencies": {
     "@angular/cli": "<%= version %>",

